Sources/Commands/SwiftTestTool.swift

```diff
@@ -165,7 +165,8 @@ public class SwiftTestTool: SwiftTool<TestToolOptions> {
             to: { $0.listTests = $1 })
 
         binder.bind(
-            option: parser.add(option: "--parallel", kind: Bool.self),
+            option: parser.add(option: "--parallel", kind: Bool.self,
+                usage: "Run the tests in parallel. Note: This mode of testing may cause problems for tests which are not written to be able to safely run in parallel."),
```

Comment:
I would take the verbose note out here, I think it is more appropriate for a man page or docs rather than `--help`.